package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
This file may have been inadvertently added e.g. by running an npm command in the root directory of the project

Reply:
After running `npm update`, the `agones/package-lock.json` file was generated, thought it was required but I have now deleted it. Thanks!